packages/test_base/setup.cfg

```diff
@@ -42,6 +42,7 @@ install_requires =
     pytest
     PyYAML
     requests
+    pytest-celery
```

Comment:
Since commit 1fef07cb24a40678f5e1eb37dae912dc1730ce7c we are not using `pytest-celery` any more.